packages/cms-cli/bin/cli.js

```diff
@@ -35,9 +34,16 @@ const argv = yargs
   .usage('Tools for working with the HubSpot CMS')
   .middleware([setLogLevel])
   .exitProcess(false)
-  .fail((msg, err /*, _yargs*/) => {
+  .fail((msg, err, _yargs) => {
+    // Preserve stack trace.
+    if (err) throw err;
+
     if (msg) logger.error(msg);
     if (err) logErrorInstance(err);
```

Comment:
How is this reachable if we are throwing the error on line 39?

Reply:
Hmmm, that's a good question.  I'll remove it since we are doing `logErrorInstance(err)` below.